sync_ldap_groups_to_svn_authz.py

```diff
@@ -114,7 +114,8 @@ def bind():
   """This function will bind to the LDAP instance and return an ldapobject."""
 
   ldapobject = ldap.initialize(url)
-
+  ldapobject.protocol_version = ldap.VERSION3  #paging results only apply to LDAP v3
```

Comment:
Would it make sense to turn this into a CLI option?  If you can manipulate these options as you have without affecting other AD versions or LDAP servers, it wouldn't matter but I'm not sure.

Reply:
If you want, sure i can make the ldap protocol version, into a cli option (with 3 being the default)

Reply:
LDAP 3 has existed since 1997, and LDAP 2 officially died in 2003. That's a while back... So I assert it's not worth the added complexity to add an LDAP version parameter.